Desktop/widgets/boundcontroltableview.cpp

```diff
@@ -136,11 +136,11 @@ Json::Value BoundControlTableView::_defaultValue()
 	JASPControl::ItemType itemType = _tableView->itemType();
 
 	if (itemType == JASPControl::ItemType::Double)
-		defaultValue = _tableView->defaultEmptyValue().toDouble();
+		defaultValue = _tableView->defaultValue().toDouble();
```

Comment:
Shouldnt there be a tryCatch or something around these conversions? Or a check if it *can* be converted?

Reply:
toDouble does not throw an error, it just returns 0 if it cannot convert, which is good enough.